src/reformatters/contrib/uarizona/swann/analysis/dynamical_dataset.py

```diff
@@ -37,7 +37,7 @@ def validators(self) -> Sequence[validation.DataValidator]:
     def operational_kubernetes_resources(self, image_tag: str) -> Iterable[CronJob]:
         operational_update_cron_job = ReformatCronJob(
             name=f"{self.dataset_id}-operational-update",
-            schedule="0 0 * * *",
+            schedule="0 22 * * *",
```

Comment:
Looking at the Last Modified headers when downloading a file (and the html listings, but those are in AZ time) the files are published right before 15:05 UTC every day, with one exception where it took until 16:15 UTC. Giving 15 more mins buffer,

```suggestion
            schedule="30 16 * * *",
```